packages/swarm-docs/src/pages/Radio.mdx

```diff
@@ -1,12 +1,53 @@
-import { Radio } from '@meetup/swarm-components'
-import Example from '../components/radioExample'
+import { Radio } from '@meetup/swarm-components';
+import Example from '../components/radioExample';
+import PropsTable from '../components/propstable';
 
-# Radio
+## Radio
+---
+A wrapper for an html `<input type="radio">`.
+
+<br/>
+<br/>
+
+### Snippet
+---
+
+Use this interactive snippet to try the component:
+<br/>
 
 ```.jsx
-<Radio label="swarm" checked={1 == 1} />
+<Radio name="swarm" label="swarm" value="1" checked={1 == 1} />
```

Comment:
Am I missing something? Why is `checked={1 == 1}` used as an example? I feel it's confusing as an example for the `boolean` prop.

Reply:
I didn't write this example, but I think it's supposed to encourage people to play around with the snippet. I can change it if you feel strongly about it.

Reply:
Yep, sorry, didn't realize it's not what you changed when I was writing it.

Let's have a separate ticket for addressing these - I feel strongly that it doesn't encourage me to do anything, but confuses me even more. If this had some variable in it that is compared, it'd be useful, but as is it looks like a computer-sciencey tricky notation that I need a degree to understand.